unittests/framework/platforms/node/raid/test_raid.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import os
+import sys
+import time
+import unittest
+from unittest.mock import Mock, patch
+from cortx.utils.process import SimpleProcess
+
+PROJECT_ROOT = "/".join(os.path.abspath(__file__).split("/")
+                        [:-6]) + "/low-level"
+sys.path.append(PROJECT_ROOT)
+from framework.platforms.node.raid.raid import RaidArray
+
+
+class TestServiceMonitor(unittest.TestCase):
+
+    def setUp(self):
+        self.raid_array = RaidArray()
+        mdadm = '# mdadm.conf written out by anaconda\nMAILADDR root\nAUTO +imsm +1.x -all\nARRAY /dev/md/0 level=raid1 num-devices=2 UUID=6c9cbe03:018e1941:5b6fff24:92796749\nARRAY /dev/md/1 level=raid1 num-devices=2 UUID=bcf899b5:17b8cdc9:9ed1e528:5148c4ad\n'
```

Comment:
keep in seperate file itself, why to paste here

Reply:
This is mock response, why we need to separate it form test file?

Reply:
Its easier to view & relate in separate file as conf

Reply:
Done